solidity-plugin/src/main/java/org/sonarsource/solidity/CognitiveComplexityVisitor.java

```diff
@@ -36,42 +36,46 @@ public Token visitFunctionDefinition(FunctionDefinitionContext ctx) {
   public Token visitForStatement(ForStatementContext ctx) {
     nestingLevel++;
     complexity += nestingLevel;
-    System.out.println("For: " + complexity + " nestingLevel: " + nestingLevel);
+    // System.out.println("For: " + complexity + " nestingLevel: " + nestingLevel);
     super.visitForStatement(ctx);
     nestingLevel--;
-    System.out.println("Leaving For Loop " + nestingLevel);
+    // System.out.println("Leaving For Loop " + nestingLevel);
     return null;
   }
 
   @Override
   public Token visitWhileStatement(WhileStatementContext ctx) {
     nestingLevel++;
     complexity += nestingLevel;
-    System.out.println("While: " + complexity + " nestingLevel: " + nestingLevel);
+    // System.out.println("While: " + complexity + " nestingLevel: " + nestingLevel);
     checkExpressionIncrementsComplexity(ctx.expression());
     super.visitWhileStatement(ctx);
     nestingLevel--;
-    System.out.println("Leaving While Loop " + nestingLevel);
+    // System.out.println("Leaving While Loop " + nestingLevel);
     return null;
   }
 
   @Override
   public Token visitDoWhileStatement(DoWhileStatementContext ctx) {
     nestingLevel++;
     complexity += nestingLevel;
-    System.out.println("DoWhile: " + complexity + " nestinglEvel: " + nestingLevel);
+    // System.out.println("DoWhile: " + complexity + " nestinglEvel: " + nestingLevel);
     checkExpressionIncrementsComplexity(ctx.expression());
     super.visitDoWhileStatement(ctx);
     nestingLevel--;
-    System.out.println("Leaving DoWhile Loop " + nestingLevel);
+    // System.out.println("Leaving DoWhile Loop " + nestingLevel);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=21&issues=AWQn3a8JYwZU9vvrp9dL&open=AWQn3a8JYwZU9vvrp9dL)

[ISSUE_KEY]: AWQn3a8JYwZU9vvrp9dL